### ISSUE TITLE
CROUTON Kali doesn't start

### ISSUE THREAD

**Issue by Usr00** [dnschneid/crouton#2949]
```
name: kali
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/kali...
crouton: version 1-20161129162558~master:5f2f11f9
release: kali-rolling
architecture: amd64
xmethod: xiwi
targets: xfce,xiwi
host: version 88772.70.0 (Official Build) stable-channel link
kernel: Linux localhost 3.8.11 #1 SMP Wed Dec 7 23:29:39 PST 2016 x86_64 GNU/Linux
freon: yes
```

#### Please describe your issue:
I installed Kali Linux on my Chromebook Pixel 2013. Everything worked fine. 
After i rebooted my laptop I was about to start kali but it doesn't start.
Anyone has a fix for this? Thank you :)

Note -- the only thing I did while I was inside Kali terminal was to run these commands
sudo apt-get update
sudo apt-get upgrade 
sudo apt-get install kali-linux-full

#### If known, describe the steps to reproduce the issue:
Basically when I go to enter the command : sudo startxfce4 
this is what I get: 

Entering /mnt/stateful_partition/crouton/chroots/kali...
/usr/bin/startxfce4: Starting X server

/usr/lib/xorg/Xorg.wrap: Only console users are allowed to run the X server
/usr/bin/xinit: giving up
/usr/bin/xinit: unable to connect to X server: Connection refused
/usr/bin/xinit: server error
Unmounting /mnt/stateful_partition/crouton/chroots/kali...
Sending SIGTERM to processes under /mnt/stateful_partition/crouton/chroots/kali...

**Comment by pcmschool** [comment]
This video can help you: https://www.youtube.com/watch?v=Rh3q5jj3y4k

**Comment by dnschneid** [comment]
You'll need to update your chroot to fix X11.
